src/interp.rs

```diff
@@ -106,6 +115,14 @@ impl<'tcx> TyVal<'tcx> {
                 },
                 _ => unimplemented!()
             },
+            TypeVariants::TyBool => {
+                let mut val = false;
```

Comment:
This can just be `assert_eq!(...); let val = bytes[0] == 1`.

Reply:
Fixed